subcommand/server-acl-init/command.go

```diff
@@ -73,8 +81,10 @@ func (c *Command) init() {
 		"Toggle for creating a client agent token")
 	c.flags.BoolVar(&c.flagCreateSyncToken, "create-sync-token", false,
 		"Toggle for creating a catalog sync token")
+	c.flags.BoolVar(&c.flagCreateInjectorToken, "create-injector-namespace-token", false,
+		"Toggle for creating a connect injector token. Only required when namespaces are enabled")
 	c.flags.BoolVar(&c.flagCreateInjectAuthMethod, "create-inject-token", false,
```

Comment:
now that we have two inject flags, what do you think about creating a new `-create-inject-auth-method` flag and deprecating this one? That will make it clearer.
```go
	c.flags.BoolVar(&c.flagCreateInjectorToken, "create-inject-namespace-token", false,
		"Toggle for creating a connect injector token. Only required when namespaces are enabled")
	c.flags.BoolVar(&c.flagCreateInjectAuthMethod, "create-inject-auth-method", false,
		"Toggle for creating a connect inject auth method")
	c.flags.BoolVar(&c.flagCreateInjectAuthMethod, "create-inject-token", false,
		"Toggle for creating a connect inject auth method. Deprecated: use -create-inject-auth-method instead.")

```

Reply:
This is fine if folks only define one of the flags. If they define both, they may get unexpected results.

Reply:
True. You could then have a new variable and then error out if they're both set. Luckily it's unlikely because 99.9% of people are using the helm chart and would never get in that situation.